### PR TITLE
fix CI

### PR DESCRIPTION
**Pull request by haircommander**
finally..

### CODE REVIEW

**Comment by haircommander** [comment]
currently failing:
```

2022-01-21T20:36:21.0739606Z 
2022-01-21T20:36:21.0748094Z [Fail] Podman run networking [It] podman run network expose host port 18081 to container port 8000 using rootlesskit port handler 
2022-01-21T20:36:21.0748735Z /home/runner/work/conmon/conmon/podman/test/e2e/run_networking_test.go:390
2022-01-21T20:36:21.0748990Z 
2022-01-21T20:36:21.0749352Z [Fail] Podman run [It] podman run with cgroups=disabled runs without cgroups 
2022-01-21T20:36:21.0749802Z /home/runner/work/conmon/conmon/podman/test/e2e/run_test.go:1424
2022-01-21T20:36:21.0750052Z 
2022-01-21T20:36:21.0750464Z [Fail] Podman privileged container tests [It] podman privileged CapEff 
2022-01-21T20:36:21.0750946Z /home/runner/work/conmon/conmon/podman/test/e2e/run_privileged_test.go:35
2022-01-21T20:36:21.0751196Z 
2022-01-21T20:36:21.0751854Z [Fail] Podman privileged container tests [It] podman cap-add CapEff 
2022-01-21T20:36:21.0752325Z /home/runner/work/conmon/conmon/podman/test/e2e/run_privileged_test.go:35
2022-01-21T20:36:21.0752572Z 
2022-01-21T20:36:21.0753064Z [Fail] Podman privileged container tests [It] podman cap-add CapEff with --user 
2022-01-21T20:36:21.0753535Z /home/runner/work/conmon/conmon/podman/test/e2e/run_privileged_test.go:35
2022-01-21T20:36:21.0753777Z 
```
which is good enough for me

@rhatdan @TomSweeneyRedHat @mheon @giuseppe @saschagrunert PTAL

**Review by giuseppe — APPROVED** [review]
LGTM

**Review by saschagrunert — APPROVED** [review]
/lgtm

File: hack/github-actions-setup [hunk 1]
```diff
@@ -132,4 +133,10 @@ install_testdeps() {
     popd
 }
 
+setup_etc_subid() {
+	echo "containers:200000:65536" | sudo tee -a /etc/subuid
+	echo "containers:200000:65536" | sudo tee -a /etc/subgid
+
```
Comment (saschagrunert) [inline hack/github-actions-setup:139]: nit
```suggestion
```

**Pull request merged**